makefile

```diff
@@ -18,6 +18,10 @@
 # Common macros
 #===============================================================================
 
+ifeq ($(PLAT),)
+    PLAT:=$(shell bash dev/make/identify_os.sh)
```

Comment:
will this work on windows?

Reply:
yes, msys2/bash is requirement for building on Windows and it returns correct PLAT